Starting to write DAO tests.

kmp3325 opened the pull request:
@tuttleb Figured out a way to get DAO tests going.  Pretty simple solution.

kmp3325 committed:
Starting to write DAO tests.

tuttleb commented on src/test/java/accounting/data/TransactionDAOTest.java:
```diff
@@ -1,5 +1,23 @@
+
+    @Before
+    public void setup() {
+        DBI dbi = new DBI("jdbc:mysql://localhost:3306/accounting", "root", "root");
```
Does this mean tests are running against the database? Should we have a test database for this?

kmp3325 replied:
We could create an accounting-test database and just clear it every time...idk, I've been trying to find more elegant solutions, but there's too much involved in all of them.

kmp3325 committed:
Added an investment endpoint and made a test database for testing the DAOs.

kmp3325 commented:
We should merge this in soon because this gives us the investment endpoint.  I also changed the user to be the accounting and accounting_test user and updated the DAO test to user the test database.

tuttleb commented:
This looks okay, I wish we had a better way to manage the test database. I think more coverage is worth the tradeoff

kmp3325 committed:
All tests work.
